### ISSUE TITLE
Error when building GenerateCLPLauncher on latest master

### ISSUE THREAD

**Issue by zachmullen** [Slicer/SlicerExecutionModel#81]
```
[100%] Building C object GenerateCLP/CMakeFiles/GenerateCLPLauncher.dir/GenerateCLPLauncher.c.o
/Users/zach/dev/covalic-score-build/SlicerExecutionModel-Build/GenerateCLP/GenerateCLPLauncher.c:15:4: error: expected identifier or '('
   "/Users/zach/dev/covalic-score-build/SlicerExecutionModel-Build/ModuleDescriptionParser/bin" CONFIG_DIR_POST \
   ^
In file included from /Users/zach/dev/covalic-score-build/SlicerExecutionModel-Build/GenerateCLP/GenerateCLPLauncher.c:29:
/Users/zach/dev/covalic-score-build/Insight-Build/Modules/ThirdParty/KWSys/src/itksys/SharedForward.h:720:76: error: expected expression
  static const char* search_path_build[] = {KWSYS_SHARED_FORWARD_PATH_BUILD, 0};
                                                                           ^
/Users/zach/dev/covalic-score-build/Insight-Build/Modules/ThirdParty/KWSys/src/itksys/SharedForward.h:721:80: error: expected expression
  static const char* search_path_install[] = {KWSYS_SHARED_FORWARD_PATH_INSTALL, 0};
```

ping @vovythevov @jcfr

**Comment by vovythevov** [comment]
@zachmullen, @jcfr: PR #82 should fix the problem.

**Comment by vovythevov** [comment]
@zachmullen: Thanks for the report, this should be fixed with #82. I'll let you close the issue once you confirm.

**Comment by zachmullen** [comment]
Yep, fixed!
